tests/post/features/log_accessible.feature

```diff
@@ -0,0 +1,5 @@
+@ci @local @post
+Feature: logs should be accessible
+    Scenario: get logs
+        Given the Kubernetes API is available
+        Then The pods logs should not be empty
```

Comment:
Minor: may be it's easier if we have standard in the sentences in steps. I suggest that all start with a lowercase as it is what we did before